### PR TITLE
SWC-3458: expose ability to create file views (show beta badge)

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
screen shot 2016-12-05 at 11 03 06 am

File: src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizard.java [hunk 1]
```diff
@@ -22,6 +24,8 @@
 	}
 	private String parentId;
 	private TableType type;
+	public static final String VIEW_HELP = "A file view is a view of all files within one or more Project or Folders.";
```
Comment (brucehoff) [inline src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizard.java]: typo:  Project -> Projects
Note:  This is a copy-paste so also fix the source.

File: src/main/resources/org/sagebionetworks/web/client/widget/entity/tabs/TablesTabViewImpl.ui.xml [hunk 2]
```diff
@@ -17,17 +17,32 @@
...
-					<b:Icon type="TABLE" pull="LEFT" addStyleNames="font-size-150" />
-					<b:Heading text="Synapse Tables" size="H4"></b:Heading>
-					<bh:Text text="Synapse Tables are designed to provide users the ability to create web-accessible, sharable, and queryable data where columns can have a user-specified structured schema."></bh:Text>
-					<bh:Div addStyleNames="margin-10">
-						<b:Button ui:field="tableLearnMoreButton" text="Learn More" type="PRIMARY"  />
-					</bh:Div>
-				</b:BlockQuote>
-			</g:FlowPanel>
-			
+			<b:Row>
```
Comment (brucehoff) [inline src/main/resources/org/sagebionetworks/web/client/widget/entity/tabs/TablesTabViewImpl.ui.xml:20]: include a beta badge

**Comment by jay-hodgson** [comment]
view help beta badge

**Pull request merged**